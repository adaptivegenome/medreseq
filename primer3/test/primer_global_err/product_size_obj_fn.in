SEQUENCE_ID=product_size_opt_not_def
PRIMER_PAIR_WT_PRODUCT_SIZE_LT=1
SEQUENCE_TEMPLATE=CAGCTGATCGATCGATCGATCGTAGCTAGCTAGCTAGCTAGCTGATCGATCGATGCTAGCTAGCTAGCTAGTCGATCGATGCTAGCTAGCTAGCTAGCTGATCGATCGTAGCTAGCTAGCTAGCTGACTGATCGATCGATGCTAGCTAGCTAGCTAGCTGATCGATCGTAGCTAGCT
=
