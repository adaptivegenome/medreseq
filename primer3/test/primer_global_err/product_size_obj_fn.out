SEQUENCE_ID=product_size_opt_not_def
PRIMER_PAIR_WT_PRODUCT_SIZE_LT=1
SEQUENCE_TEMPLATE=CAGCTGATCGATCGATCGATCGTAGCTAGCTAGCTAGCTAGCTGATCGATCGATGCTAGCTAGCTAGCTAGTCGATCGATGCTAGCTAGCTAGCTAGCTGATCGATCGTAGCTAGCTAGCTAGCTGACTGATCGATCGATGCTAGCTAGCTAGCTAGCTGATCGATCGTAGCTAGCT
PRIMER_ERROR=Product size is part of objective function while optimum size is not defined
=
